docs/docs/how-to/plugins-and-themes/creating-a-source-plugin.md

```diff
@@ -526,8 +526,9 @@ Then export a new function `onCreateNode`, and call `createRemoteFileNode` in it
 exports.onCreateNode = async ({
   node, // the node that was just created
   actions: { createNode, createNodeField },
+  store,
+  cache,
   createNodeId,
-  getCache,
```

Comment:
`getCache` should be preferred over `cache` as this should allow to use correct cache (cache of `gatsby-source-filesystem` and not one of calling plugin). Unless there is a bug and `getCache` doesn't work anymore?

Reply:
Also `store` doesn't actually seem used (anymore?) - https://github.com/gatsbyjs/gatsby/blob/ad4b8da836d219c13655f0e69f111684efb427a2/packages/gatsby-source-filesystem/src/create-remote-file-node.js#L193-L204

Reply:
Fixed. Thanks!